frontend/src/components/RandomPost.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+
+const RandomPost = props => {
+  //fetch random post
```

Comment:
Is this necessary? 🤔

Reply:
this feature will be implemented later
added TODO as discussed